its/install.ts

```diff
@@ -0,0 +1,54 @@
+import { DefinitionQueryOrder } from "azure-devops-node-api/interfaces/BuildInterfaces";
+import fs from "fs";
+import path from "path";
+import { AZDO_PIPELINE_NAME_PREFIX, AZDO_PROJECT, FIXTURES_PATH } from "./constant";
+import { createPipeline, getAzdoApi, getTemplatePipeline } from "./it/azdo";
+
+const CLEANUP_PAGINATION_LIMIT = 500;
```

Comment:
No pagination on the REST API, I assume we will never exceed 500 pipelines in this project 
![image](https://github.com/user-attachments/assets/402985fb-1885-411f-bafa-88e87fd0d0dc)

Reply:
![383964230-402985fb-1885-411f-bafa-88e87fd0d0dc](https://github.com/user-attachments/assets/17892700-3841-4262-93b8-2e13764cf4ca)

Reply:
😅